src/Argument.h

```diff
@@ -56,13 +56,11 @@ struct Argument {
      * By default, they are left unset, implying "no default, no min, no max". */
     Expr def, min, max;
 
-  int stride_multiples[4];
```

Comment:
Thinking about this more: I think you do need to do git rebase -i, so you can skip all of the commits that are redundant (basically, all of the commits from anyone not at Qualcomm, and probably all the commits making changes that were already in the "big blob merge" as you call it.